### ISSUE TITLE
error: 'no predictions found'

### ISSUE THREAD

**Issue by sriyachakravarthy** [TianjinYellow/EdgeDeviceLLMCompetition-Starting-Kit#7]
Hi! We tried evaluating the base models using the starting kit evaluation pipeline. Here are some points/issues:
1. For phi2 and llama models, we are getting 'prediction not found' error. 
2. Could we please get some idea on the time taken for inference with respect to the GPU memory and model size?

**Comment by TianjinYellow** [comment]
Hi, 

i tested the repository just now on one A100 GPU and it works well.  I did not find the errors you showed. 
<img width="1101" alt="Screenshot 2024-07-31 at 07 38 52" src="https://github.com/user-attachments/assets/9bd9855b-3ddb-401a-a5c7-8bdd69998218">

(1) please check whether data is downloaded and put it in the folder that contains the configs folder, human_eval folder. 
(2) One of the quickest ways to solve it is to uninstall the previous encompass. then git clone the repository and re-install it. 
(3) for the commonsense_qa task and phi2, it takes 27 GB GPU memory and around 15 minutes to evaluate it.

Best,

**Comment by sriyachakravarthy** [comment]
Hi! Is this the correct directory flow? 
<img width="174" alt="Screenshot 2024-07-31 123007" src="https://github.com/user-attachments/assets/fe8605c4-ee2d-4616-a7bd-c9e80009a53e">

**Comment by TianjinYellow** [comment]
Yes. Additionally, when I run the evaluation, I set this directory as the workspace as well.

**Comment by sriyachakravarthy** [comment]
> Hi,
> 
> i tested the repository just now on one A100 GPU and it works well. I did not find the errors you showed. <img alt="Screenshot 2024-07-31 at 07 38 52" width="1101" src="https://private-user-images.githubusercontent.com/70349180/353722220-9bd9855b-3ddb-401a-a5c7-8bdd69998218.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0OTI3NzQsIm5iZiI6MTcyMjQ5MjQ3NCwicGF0aCI6Ii83MDM0OTE4MC8zNTM3MjIyMjAtOWJkOTg1NWItM2RkYi00MDFhLWE1YzctOGJkZDY5OTk4MjE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODAxVDA2MDc1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4YmViMGNmZWUzNDZkMGE1YmUyMjY2ODYxOGZiNGQ5NzRkNGMyMWRiZTg5ODEzMTU4NmYyM2Q4ODU0YTQyZTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bLK9aKxhdtBoxYaXERk9vQGk2fjJ9nsIMqtRpUq-RwU">
> 
> (1) please check whether data is downloaded and put it in the folder that contains the configs folder, human_eval folder. (2) One of the quickest ways to solve it is to uninstall the previous encompass. then git clone the repository and re-install it. (3) for the commonsense_qa task and phi2, it takes 27 GB GPU memory and around 15 minutes to evaluate it.
> 
> Best,

Thank you for the help! It worked for phi2. Were tests done for other models as well? We tried on Qwen2-7B commonsenseqa_gen and we got 0% accuracy. Please confirm.
